### PR TITLE
Prepare the new error handling

### PR DESCRIPTION
**Pull request by rivanfebrian123**
Better error handling and logging

### CODE REVIEW

File: lib-core.sh [hunk 1]
```diff
@@ -481,31 +480,42 @@ full_rootify() {
 
 userify() {
   trap true SIGINT
-  sudo -u "${MY_USERNAME}" ${@} 2> "${WHITESUR_TMP_DIR}/error_log.txt" || operation_canceled
-  trap sig_c SIGINT
+  sudo -u "${MY_USERNAME}" ${@} || operation_aborted
+  trap stop_animation SIGINT
```
Comment (rivanfebrian123) [inline lib-core.sh:484]: Forgot to change this to "signal_exit"

File: lib-core.sh [hunk 2]
```diff
@@ -460,16 +464,11 @@ remind_relative_path() {
...
-  rm -rf "${WHITESUR_TMP_DIR}"
-}; export -f lockWhiteSur
-
 rootify() {
   trap true SIGINT
   prompt -w "Executing '$(echo "${@}" | cut -c -35 )...' as root"
-  sudo ${@} 2> "${WHITESUR_TMP_DIR}/error_log.txt" || operation_canceled
-  trap sig_c SIGINT
+  sudo ${@} || operation_aborted
+  trap stop_animation SIGINT
```
Comment (rivanfebrian123) [inline lib-core.sh:471]: This too

**Comment by rivanfebrian123** [comment]
Oh my God, I'm not finished yet. I thought a draft PR is unmergeble

**Comment by vinceliuice** [comment]
As long as it can be installed, it’s my fault, I didn’t see clearly

**Pull request merged**